README.md

```diff
@@ -890,3 +900,11 @@ Note that if the change includes adding new statistics you might need to introdu
 * `mix dialyzer` to run dialyzer for type checking, might take a while on the first invocation (try building plts first with `mix dialyzer --plt`)
 * `mix credo --strict` to find code style problems
 * or run `mix guard` to run all of them continuously on file change
+
+
+## Copyright and License
+
+Copyright (c) 2016 Tobias Pfeiffer
```

Comment:
Does.. this give me anything? :sweat_smile: I don't know Copyright too well, to my eyes it seems unnecessary but always happy to learn :)

Reply:
IANAL, but this is how most software license term is applied in software project. For example, [Apache-2 license usage](https://apache.org/licenses/LICENSE-2.0#apply).